### PR TITLE
add Tinti et al. to reproducible research

### PR DESCRIPTION
**Pull request by daisy20170101**
add Norcia event

### CODE REVIEW

**Review by sebwolf-de — APPROVED** [review]

File: Documentation/reproducible-research.rst [hunk 1]
```diff
@@ -115,6 +115,14 @@ A non-exhaustive list is given below:
...
      - `Taufiqurrahman et al. (2022) <https://doi.org/10.1002/essoar.10510965.1>`_
      - `zenodo.org/record/6386938 <https://zenodo.org/record/6386938>`_
+     
+   * - 2016 Mw 6.5 Norcia |br| earthquake
+     - dynamic
+     - N
+     - LSW
+     - 1
+     - `Tinti et al. (2021) <https://doi.org/10.1016/j.epsl.2021.117237>`_
+     - `github repository <https://github.com/git-taufiq/NorciaMultiFault>`_
```
Comment (sebwolf-de) [inline Documentation/reproducible-research.rst:125]: Just a technical note: Maybe, we should link to the latest commit (https://github.com/git-taufiq/NorciaMultiFault/tree/6675035). After all git repositories can change...

Comment (daisy20170101) [reply Documentation/reproducible-research.rst:125]: This comes along with the model in the paper published. I think no one will update it. Or we can put a release version?

Comment (krenzland) [reply Documentation/reproducible-research.rst:125]: You can define a tag/release: https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

Comment (Thomas-Ulrich) [reply Documentation/reproducible-research.rst:125]: I would keep it as it is. If someone updates the repository, it means it is needed, and hopefully the new state is better than the older one.

**Review by Thomas-Ulrich — APPROVED** [review]

**Pull request merged**